sync_first/sync_first_app/views.py

```diff
@@ -1,6 +1,56 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
+from .models import Incident, Person
 
 from django.shortcuts import render
+from django.http import HttpResponse,Http404
 
 # Create your views here.
+def show_new_incident_page(request):
+    return render(request, 'new_incident.html')
+
+def all_incidents(request):
+    context = {
+        "incidents" : Incident.objects.all()
+    }
+    return render(request, "all_incidents.html", context)
+
+def get_incident_by_id(request, incident_id):    
+    try:
+        context = {
+            "incident": Incident.objects.get(pk=incident_id)
+        }
+    except Incident.DoesNotExist:
+        raise Http404("Incident not found.")
+
+    return render(request, "incident.html", context)
+
+
+def show_search_page(request):
+    return render(request, 'search.html')
+
+def get_incidents_by_person_id(request, person_id_number):
+    incidents_lists = []
+    try:
+        incidents_lists.append(Incident.objects.get(main_person = person_id_number))
```

Comment:
get expects only one object to be returned

Reply:
# TODO statement added to code and later be fixed.